cypress/integration/login.js

```diff
@@ -8,20 +8,21 @@ describe('Sourcebooks login', function() {
         cy.get('.Select.not-valid').should('be.visible')
     })
 
-    it('Should be able to login with role User', function () {
+    const roles = ['User', 'Team Lead', 'Manager', 'Accountant', 'Admin'];
+    const menu = ['1', '2', '5', '5', '6'];
+    roles.forEach((role, id) => {

```

Comment:
you also use id in this loop, but it;s not actually used in test. Also this id would always be 0